src/accessibility/accessibility.js

```diff
@@ -52,6 +58,12 @@ Player.provide('accessibility',
                     Player.set('qualityMenuExpanded', true)
                     $(document.activeElement).attr("aria-label", Player.translate("expanded"))
                   }
+                  if($(document.activeElement).hasClass('subtitle-button')){
+                    Player.set('subtitleMenuExpanded', true)
+                    var subtitleOn = Player.get('subtitleLocale') != "";
+                    if(subtitleOn) $(document.activeElement).attr("aria-label", Player.translate("subtitle_on_expanded"))
+                    else $(document.activeElement).attr("aria-label", Player.translate("subtitle_off_expanded"))
```

Comment:
I guess this is equivalent:

```suggestion
    $(document.activeElement).attr("aria-label", Player.translate( Player.get('subtitleLocale') != "" ? "subtitle_on_expanded" : "subtitle_off_expanded"))
```